reframework/autorun/NoKillCam+SlowMo.lua

```diff
@@ -380,6 +375,8 @@ local function PreRequestCamChange(args)
 			else
 				return;
 			end
+		elseif isSlowMo and endFlow <= 1 and endCapture == 2 then
```

Comment:
This confuses me greatly but I'll take your word that it does what you say it does

Reply:
It also might be possible with this that sometimes the endflow is like 1 or something right when you kill the monster which might be why for some people the slowmo and skip doesnt activate on occasion but it's yet to be proven, either way it probabably wouldnt hurt if the initial check was endflow 1 instead of 0.